unit-registry/core/src/main/java/org/openbase/bco/registry/unit/core/UnitRegistryController.java

```diff
@@ -507,6 +537,37 @@ public void registerMethods(final RSBLocalServer server) throws CouldNotPerformE
     }
 
     @Override
+    public Future<AuthenticatedValue> registerUnitConfig(final AuthenticatedValue authenticatedValue) throws CouldNotPerformException {
```

Comment:
refactor (and all others as well) to  registerUnitConfigAuthenticated to be api stable